tests/gis_tests/gis_migrations/test_operations.py

```diff
@@ -214,6 +228,98 @@ def test_create_model_spatial_index(self):
         if connection.features.supports_raster:
             self.assertSpatialIndexExists("gis_neighborhood", "rast", raster=True)
 
+    @skipUnlessDBFeature("can_alter_geometry_field")
+    def test_alter_field_remove_spatial_index(self):
+        """
+        Test that indexes are removed when `spatial_index` is changed to False
+        """
```

Comment:
To add more explanation, docstrings should state the expected behavior and omit prefixes like "Test that" since all tests test things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).